prometheus-wireguard-exporter/riscv64.patch

```diff
@@ -0,0 +1,32 @@
+diff --git repos/community-x86_64/PKGBUILD repos/community-x86_64/PKGBUILD
+index a573d5d..2528e6d 100644
+--- repos/community-x86_64/PKGBUILD
++++ repos/community-x86_64/PKGBUILD
+@@ -23,7 +23,26 @@ sha512sums=('9ff42deaf28a45be26cf0211b11878be873467bee0bdb128efec3c1a9d4f7214f53
+ 
+ prepare() {
+   cd "prometheus_wireguard_exporter-${pkgver}"
+-  cargo fetch --locked --target "$CARCH-unknown-linux-gnu"
++
++  # RISC-V Patches:
++  # rustc(1) recognizes `riscv64gc' instead of `riscv64'. Thus,
++  # we need to manually substitute riscv64 with riscv64gc in 
++  # --target. The `--target' argument, according to Arch Wiki,
++  # "tells cargo to only fetch dependencies needed for the
++  # specific target platform being built, thus reducing downloads."
++  # Ref: https://wiki.archlinux.org/title/Rust_package_guidelines#Prepare
++  local _target="$CARCH-unknown-linux-gnu"
++  if test $CARCH = "riscv64"; then
```

Comment:
As Xeonacid has suggested, we can simply remove this if branch and do necessary things unconditionally.

Reply:
Thank you. I will fix it right now.

Reply:
How about the `ring` patch? May I do it unconditionally as well?

Reply:
I think so, at least we are doing so previously

Reply:
For this part we are looking forward to make the ring upstream package fully functional, or find an equivalent alternative, so that all patches specifically for ring can be removed, which would greatly reduce the amount of patched packages.

IMO maybe when some days we finally give up on that we will come back to the conditionally patch path.

Reply:
OK, thank you!